package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "react-native-mdm",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [
+    "react-native"
+  ],
+  "author": "Ying Hang Eng",
+  "license": "MIT",
+  "devDependencies": {
```

Comment:
Looks like we removed `peerDependencies` as part of moving `react-native` to `devDependencies`. Keep `peerDependencies`